services/net/image/ImageAction.cs

```diff
@@ -212,5 +306,29 @@ protected DateTime GetLocalDateTime(DataSourceModel dataSource, DateTime date)
     {
         return date.ToTimeZone(DataSourceIngestManager<ImageOptions>.GetTimeZone(dataSource, this.Options.TimeZone));
     }
+
+    /// <summary>
+    /// map front page image name with datasource code
+    /// </summary>
+    /// <param name="dataSource"></param>
+    /// <returns></returns>
+    private string MappingImageFile(DataSourceModel dataSource)
```

Comment:
This isn't ideal.  If we need to match a data source with an image naming convention.  Either the data source `Code` should be appropriate, or the `DataSource.Connection` should contain a relevant configuration value.